### ISSUE TITLE
Improvements for SONG

### ISSUE THREAD

**Issue by scottcain** [cancogen-virus-seq/portal#235]
From @lindaxiang: with the growing number of analysis for each project, the time which spent on getting the song analysis becomes a blocker. As the current song api endpoint does not support pagination, it kept timing out and took almost half day just to get back the song analysis (34k analysis) for a given study like LSPQ-QC. Any plan when we will get Alexis’s song performance improvement into Virusseq SONG server? I hope that would help to solve the problem.

This is presumably related to https://github.com/cancogen-virus-seq/portal/issues/222

**Comment by b-f-chan** [comment]
@sifavahora @UmmulkiramR couple suggestions:

(1) We should definitely upgrade to the latest SONG with pagination features before starting the work package features
(2) We should also audit the versions of other Overture components to see if any upgrade needed; feel free to create separate ticket and also consult Jon/Justin on this

**Comment by leoraba** [comment]
Virusseq is currently using the latest SONG version (v5.0.3) . 
Pagination features in SONG were implemented since v4.8.0
